src/Calliope/Concerns/BuildsQuery.ts

```diff
@@ -27,7 +27,7 @@ export type QueryParams = Partial<{
     offset: number;
     limit: number;
     page: number;
-}>;

```

Comment:
I would rather avoid string index here because a typo will not come up as an actual error, eg.: `relationExists` instead of `relationsExists`. `& Record<string, unknown>` is already included in some places where the user might pass in unknown keys.
This type could always be extended with module augmentation.